### ISSUE TITLE
About the Synthetic Repetition

### ISSUE THREAD

**Issue by Cecilia-yu412** [google-research/google-research#1973]
Hi @debidatta，i'm trying to figure out how the period length labels are generated when training RepNet, . As it described in the paper,"_Then, we sample a clip C of random length P frames from V. This clip C is repeated K times (where K > 1) to simulate videos with repetitions._"Is there anything worth noting when choosing the length of clip C and doing the inversion and splicing to ensure good training results? For example, the way of generating random numbers and the choice of the number of repetitions, I hope your answer.

**Comment by debidatta** [comment]
Yes you should ensure that the chosen segment gets repeated more than once. For example if you chose a segment of 32 frames or less it can repeat at least 2 times for the video length of 64. If you choose a longer segment it cannot repeat at least 2 times.

**Comment by Cecilia-yu412** [comment]
> Yes you should ensure that the chosen segment gets repeated more than once. For example if you chose a segment of 32 frames or less it can repeat at least 2 times for the video length of 64. If you choose a longer segment it cannot repeat at least 2 times.

Thank you @debidatta, I got that. I have two other doubts, is it necessary to ensure that the number of duplicate clips of each synthetic video  matches some distribution between 2~32 to prevent overfitting, or  randint() has been sufficient? The second one is whether a large number of training sets are required for training? I want to synthesize the data on a training set of number about 800, roughly how many training samples should be generated， should i use a video for more than one time?
